src/modal/modal.js

```diff
@@ -251,6 +251,10 @@ angular.module('ui.bootstrap.modal', [])
       }
 
       $document.bind('keydown', function (evt) {
+        if (evt.isPropagationStopped() || /^(?:input|textarea|select)$/i.test(evt.target.nodeName)) {
```

Comment:
Is this supposed to be `evt.defaultPrevented`? https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented